index.html

```diff
@@ -33,12 +33,18 @@
                     company:    "1st edition, JustSystems; 2nd edition, Invited Expert" },
                 {   name:       "<span lang='ja'>小林 敏</span> (Toshi Kobayashi)",
                     company:    "Invited Expert" },
+                {   name:       "Nathaniel McCully",
+                    company:    "Adobe" },
                 {   name:       "<span lang='ja'>小野澤 賢三</span> (Kenzou Onozawa)",
                     company:    "Invited Expert" },
                 {   name:       "Felix Sasaki",
                     company:    "1st edition, University of Applied Sciences Potsdam; 2nd edition, DFKI GmbH" },
+                {   name:       "<span lang='ja'>下農 淳司</span> (Atsushi Shimono)",
+                    company:    "W3C" },
                 {   name:       "<span lang='ja'>塩澤 元</span> (Hajime Shiozawa), (<span lang='ja'>第２版</span> 2nd edition)",
                     company:    "Invited Expert" },
+                {   name:       "<span lang='ja'>薛 富僑</span> (Fuqiao Xue)",
```

Comment:
This should have lang="zh-hans".

Reply:
actually, I'm not sure it is really -hans, since the last character seems not one for -hans...
https://ja.wiktionary.org/wiki/%E5%83%91

Reply:
I found three presentation in self introduction page of @xfq , and it seems that is 'ja' version. @r12a, should we choose one in one's native language (seems so,,)?
https://www.w3.org/People/Xue/

Reply:
I think we should ask @xfq.  Normally, yes, it would be in the person's native script.

Reply:
My preference is to use `ja` (Kanji) in jlreq and use `zh-hans` (Simplified Chinese characters) in clreq.

If you feel that I need to use the native script (`zh-hans`) in a unified manner, you can copy my name from clreq. Thank you!

Reply:
@r12a @xfq , then, let me propose to go as current version.

Reply:
A bit unusual, but ok by me.

Reply:
thank you. let me close this as resolved.